README-dnssec.md

```diff
@@ -0,0 +1,296 @@
+# DNSSEC setup for dnsseed
+
+##### Table of Contents  
+[Requirements](#Requirements)  
+[Software needed](#Software)  
+[Setup](#Setup)  
+[Testing](#Testing)  
+[Example](#Example)  
+[Improvements](#Improvements)  
+[Links](#Links)  
+
+
+<a name="Requirements"/>
+
+## Requirements
+* run only this service on the host (security precaution)
+
+<a name="Software"/>
+
+## Software needed
+
+* tor (if you like to seed tor nodes)
```

Comment:
If you put the "if" here, people may ignore it and it will break when you use port 9050 as it may not be obvious to users that the use of 9050 implies tor. So i'd suggest just saying that tor is required for this document

Reply:
I agree.